source/jsonnet/england-wales/ccs/blocks/individual/ethnic_group_black.jsonnet

```diff
@@ -42,24 +46,36 @@ local question(title) = {
   ],
 };
 
-local nonProxyTitle = 'Which one best describes your Black, Black British, Caribbean or African ethnic group or background?';
-local proxyTitle = {
+local nonProxyEnglandTitle = 'Which one best describes your Black, Black British, Caribbean or African ethnic group or background?';
+local proxyEnglandTitle = {
   text: 'Which one best describes <em>{person_name_possessive}</em> Black, Black British, Caribbean or African ethnic group or background?',
   placeholders: [
     placeholders.personNamePossessive,
   ],
 };
+local nonProxyWalesTitle = 'Which one best describes your Black, Black Welsh, Black British, Caribbean or African ethnic group or background?';
```

Comment:
For the Welsh variant, should the third answer option for this question also be updated to include `black welsh` as it does in the HH/HI schema?

Reply:
The requirement is to have it without on CCS - probably because it will be interviewer led.